agent/util-scripts/pbench-register-tool

```diff
@@ -215,6 +215,8 @@ while true; do
 	esac
 done
 
+check_install_rpm "pbench-sysstat"
```

Comment:
This is both too much and too little: we want to check that the tool is installed, not the RPM. But we need to check on every node where tools are registered. The trouble is that `pbench-register-tool` is only invoked on the controller. The remotes get their marching order from the Tool Meister, so they cannot even check until the run has been started. We need a `sanity-check` Tool Meister operation I think: every run needs to start with it and abort if it fails.